services/openai_analysis_service/app.py

```diff
@@ -273,11 +273,14 @@ def run_openai_summarization(data_dict):
         )
         pinecone_index = pinecone.Index(summarize_embedding_index)
         metadata = {}
-        if z is not None:
-            metadata["redshift"] = z
+        metadata["redshift"] = z if z is not None else -1
```

Comment:
Is this strictly necessary? Should it only be used in certain circumstances?

Reply:
I always want to have redshift in the metadata but might be able to get away without it.

Reply:
I guess my worry is whether there will be confusion about a negative default redshift.